README.md

```diff
@@ -63,6 +68,12 @@ Call `movieArt(function(e){console.log(e);});` or run the CLI command with no ar
 
```

Comment:
Need to inform users that year is only required for movies (i think). Is size allowed for TV? I can check api, just wondering off the top of your head.